packages/css/src/components/context/context.template.ts

```diff
@@ -0,0 +1,51 @@
+import { Context } from '@dso-toolkit/sources';
+import { html, nothing } from 'lit-html';
+import { buttonTemplate } from '../button/button.template';
+
+export function contextTemplate({ children, label, type }: Context) {
+  return html`
+    ${type == 'label'
+      ? html`
+        <div class="dso-context-wrapper">
+          <span class="control-label dso-context-label">
+            ${label}
+          </span>
+          <div class="dso-context-container">
+            ${children}
+            ${buttonTemplate({ url: '#', modifier: 'btn btn-tertiary', label: 'Versies', icon: { icon: 'chevron-down' } })}
+          </div>
+          <div class="row">
```

Comment:
Context is een component wat als _verrijking_ toegevoegd kan worden. De opbouw wrapper/label/container is goed, maar:

Wat hier staat moet door `children` worden geplaatst. Net zoals wat er in `context-container` komt te staan door een property moet komen. De inhoud van context is de buttonTemplate.

Reply:
Zie mijn antwoord hierboven, alle inhoud wordt nu via een `TemplateFnReturnType` geplaatst:
- Label
- Content
- Children